### PR TITLE
tested & fixed issue for API integration for upcoming events

### PR DESCRIPTION
**Pull request by avinashvidyarthi**
Improvement in upcoming events

### CODE REVIEW

File: src/environments/environment.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@
 export const environment = {
   production: false,
   // server: 'https://ooug-dev-1fbf3.el.r.appspot.com',
-  server: 'http://localhost:3000/',
+  server: 'http://localhost:3000',
```
Comment (InSantoshMahto) [inline src/environments/environment.ts:8]: use / at end **http://localhost:3000/**

**Review by InSantoshMahto — CHANGES_REQUESTED** [review]
forget about what we did earlier just use camel case for all type of data we store in collections.

```javascript
# Example

# Currently
const payload = {
Date: {month: 1, day: 1, year:2020}
}

# What We Wanted
const payload = {
date: {month: 1, day: 1, year:2020}
}
```

**Comment by InSantoshMahto** [comment]
If minor change iss required to do in backend we will do that.

**Comment by avinashvidyarthi** [comment]
Variable camelCasing done. Minor changes in backend too

**Review by InSantoshMahto — APPROVED** [review]

**Pull request merged**